README.md

```diff
@@ -16,31 +16,54 @@ The **Platform.sh CLI** is the official command-line interface for [Platform.sh]
   * [Drush](https://github.com/drush-ops/drush) (for Drupal projects)
   * Other build tools: [npm](https://www.npmjs.com/), [pip](http://docs.python-guide.org/en/latest/starting/installation/), [bundler](http://bundler.io/), etc.
 
-## Installation
+## Install
 
-Run this command to install the CLI:
+To install the CLI, use either [Homebrew](https://brew.sh/) (on Linux, macOS, or the Windows Subsystem for Linux) or [Scoop](https://scoop.sh/) (on Windows):
 
-    curl -sS https://platform.sh/cli/installer | php
+### HomeBrew
 
-In some Windows terminals you may need `php.exe` instead of `php`.
+```console
+brew install platformsh/tap/platformsh-cli
+```
+
+### Scoop
+
+```console
+scoop bucket add platformsh https://github.com/platformsh/homebrew-tap.git
+scoop install platform
+```
+
+### Manual installation
+
+For manual installation, you can also [download the latest binaries](https://github.com/platformsh/cli/releases/latest).
 
-### Installing manually
+### Legacy installer
 
-1. Download the `platform.phar` file from the
-  [latest release](https://github.com/platformsh/platformsh-cli/releases/latest).
+Run this command to install the CLI using the legacy installer, given that you have PHP already installed:
 
-2. Rename the file to `platform`, ensure it is executable, and move it into a
-  directory in your PATH.
+```console
+curl -sS https://platform.sh/cli/installer | php
+```
+
+In some Windows terminals you may need `php.exe` instead of `php`.
 
-3. Enable autocompletion and shell aliases:
+_This installation method is considered legacy, use one of the methods above instead. Starting with version 5.x, this installation method will not be supported._
```

Comment:
I'd move this just after the title. We also should make it as a clear disclaimer, e.g.:

```
> **DISCLAIMER**
> **This installation method is considered legacy and is discouraged**.
> Use one of the methods above instead. Starting with version 5.x, this installation method will not be supported anymore.
```

Reply:
I'm fine moving that just below the section title, but I wouldn't emphasize it as much. I believe putting in italics to stand out is enough.

Reply:
OK.
But we should state that it is discouraged.

Reply:
Would this be better?

https://github.com/platformsh/platformsh-cli/compare/4cb2b57b9436135410404b7b294acbcf9a846b9f..0deab5199354e27f44b7467603f5d8906a2e636f

Reply:
👍